mvc/views/tag-helpers/authoring/sample/AuthoringTagHelpers/src/AuthoringTagHelpers/Views/Home/Contact.cshtml

```diff
@@ -0,0 +1,19 @@
+﻿@{
+    ViewData["Title"] = "Contact";
+}
+<h2>@ViewData["Title"].</h2>
+<h3>@ViewData["Message"]</h3>
+
+<address>
+    One Microsoft Way<br />
+    Redmond, WA 98052<br />
+    <abbr title="Phone">P:</abbr>
+    425.555.0100
+</address>
+
+<address>
+    <strong>Support:</strong> <email>Support</email><br />
+    <strong>Marketing:</strong>Marketing<email></email>
```

Comment:
`<strong>Marketing:</strong>Marketing<email></email>` => `<strong>Marketing:</strong> <email>Marketing</email>`

(Just surrounded marketing with the email tag)